hack/00-osd-managed-cluster-config-integration.yaml.tmpl

```diff
@@ -41,10 +41,1214 @@ parameters:
 metadata:
   name: selectorsyncset-template
 objects:
+- apiVersion: hive.openshift.io/v1
```

Comment:
This block just moved up due to how the SSS's are appended now. Nothing changed with the identity SSS.